### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by glowredman**
- Update buildscript
- Convert to GTNHMixins
- General code clean-up (first with Eclipse then with IntelliJ)

### CODE REVIEW

**Review by Dream-Master — APPROVED** [review]
to much to review lets test ingame :)

**Comment by Dream-Master** [comment]
[fml-client-3.log](https://github.com/GTNewHorizons/Galacticraft/files/10054300/fml-client-3.log)

build crash @glowredman

File: src/main/resources/mixins.GalacticraftCore.early.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "required": true,
+    "minVersion": "0.8.3-GTNH",
+    "package": "micdoodle8.mods.galacticraft.core.mixins",
+    "refmap": "mixins.hodgepodge.refmap.json",
```
Comment (mitchej123) [inline src/main/resources/mixins.GalacticraftCore.early.json]: this should be `GalacticraftCore` @glowredman

Comment (glowredman) [reply src/main/resources/mixins.GalacticraftCore.early.json]: Fixed in https://github.com/GTNewHorizons/Galacticraft/tree/fix-mixin-exception

**Pull request merged**